resources/typemappings/GenomeAndAssembly.yaml.example

```diff
@@ -1,23 +1,28 @@
-# This is an example file that demonstrates how to specify storage-to-search type mappings beyond
+# This is an example file that demonstrates how to specify "storage-to-search" type mappings beyond
 # those embedded in the type transformation files (in the storage-type and storage-object-type
-# parameters).
+# parameters). Here, "storage" implies the data source storage system (mongodb for example) and "search"
```

Comment:
Not sure if mongodb is the best example here - I doubt any data source will expose mongo directly. The workspace is probably the best example we have right now.

Reply:
Fixed.